charts/country-risk/Chart.yaml

```diff
@@ -20,7 +20,7 @@
 apiVersion: v2
 name: country-risk
 type: application
-version: 3.0.9
+version: 3.0.10
```

Comment:
Not sure, if you really have to release the Chart, just because of the copyright year change

Reply:
Good point, i will downgrade the version , but it will show as failed lint test